src/org/freedesktop/gstreamer/glib/GObject.java

```diff
@@ -588,7 +588,7 @@ private static void transform(Object data, GType type, GValue dst) {
         protected GCallback(NativeLong id, Callback cb) {
             this.id = id != null ? id : new NativeLong(0);
             this.cb = cb;
-            this.connected = this.id.intValue() != 0;
+            this.connected = this.id.longValue() != 0;
```

Comment:
Looks good. Was this actually causing any issue?

Reply:
No, just Optimization along the way